packages/wallet-sdk/src/sign/scw/SCWSigner.ts

```diff
@@ -7,12 +7,12 @@ import { RPCRequestMessage, RPCResponseMessage } from ':core/message/RPCMessage.
 import { RPCResponse } from ':core/message/RPCResponse.js';
 import { AppMetadata, ProviderEventCallback, RequestArguments } from ':core/provider/interface.js';
 import { FetchPermissionsResponse } from ':core/rpc/coinbase_fetchSpendPermissions.js';
-import { WalletConnectResponse } from ':core/rpc/wallet_connect.js';
+import { WalletConnectRequest, WalletConnectResponse } from ':core/rpc/wallet_connect.js';
 import { Address } from ':core/type/index.js';
 import { ensureIntNumber, hexStringFromNumber } from ':core/type/util.js';
 import { SDKChain, createClients, getClient } from ':store/chain-clients/utils.js';
-import { store } from ':store/store.js';
-import { assertPresence } from ':util/assertPresence.js';
+import { store, subAccountsConfig } from ':store/store.js';
```

Comment:
Importing this here creates variable shadowing. We should just access the config off of the store object. See line 100

Reply:
good catch ty. fixed